tests/test_export.py

```diff
@@ -426,3 +426,33 @@ def test__export_archive_with_evil_command(mocker):
         ],
         stderr=-2,
     )
+
+
+def test__export_archive_success_on_empty_return_value(mocker):
+    """
+    Ensure an error is not raised when qrexec call returns empty string,
+    (success state for `disk`, `print`, `printer-test`).
+
+    When export behaviour changes so that all success states return a status
+    string, this test will no longer pass and should be rewritten.
+    """
+    export = Export()
+    check_output = mocker.patch("subprocess.check_output", return_value=b"")
```

Comment:
Note that `return_value=b'   \t \n  '` would also pass.

Reply:
Replied in https://github.com/freedomofpress/securedrop-client/pull/1596#discussion_r1028556760 :+1:

Reply:
Yup, indeed - it's not ideal, but it's not a regression and it will soon be deprecated